source/frontend/src/features/acquisition/list/AcquisitionListView.tsx

```diff
@@ -48,6 +54,23 @@ export const AcquisitionListView: React.FunctionComponent<
     'No matching results can be found. Try widening your search criteria.',
   );
 
+  const { exportAcquisitionFiles } = useAcquisitionFileExport();
+
+  /**
+   * @param {'csv' | 'excel'} accept Whether the fetch is for type of CSV or EXCEL
+   */
```

Comment:
I think there was a previous comment about removing all references to CSV

Reply:
removed